### ISSUE TITLE
CLI input and output

### ISSUE THREAD

**Issue by dettbarn** [dettbarn/music-recommendations#2]
It should be possible to just call the program from the command line, without having to edit the input file by hand

The user can call musicrecs and enter the input artists either directly (in the command) or in an editor that opens automatically. (It suffices if one of the two options is implemented.)

**Comment by Vishvam10** [comment]
I can work on the CLI part ( where the user can enter the input artists along with the weights directly on the Command Line ). @dettbarn Please assign this to me.

**Comment by Vishvam10** [comment]
Hi @dettbarn ! Thanks for assigning this issue to me. 

I'll be adding command line flags to indicate that the input will be taken from the CLI. 
Do you want the input to be single-lined ( comma + space separated ) ? Something like this :

```
python main.py -i "The Beatles%3, Radiohead%2, Muse%1.5"
```
or multi-lined like this :
```
python main.py -i
> Enter the data :
>   The Beatles%3
>   Radiohead%2
>   Muse%1.5
. . .
```

**Comment by dettbarn** [comment]
Hi @Vishvam10 , that doesn't really matter to me :) whichever is easier to implement (perhaps single-lined is more difficult because artist names might contain commas themselves)

**Comment by dettbarn** [comment]
This was implemented by #3.
